exporters/otlp/otlptrace/otlptracegrpc/internal/otlptracetest/collector.go

```diff
@@ -99,8 +99,14 @@ func resourceString(res *resourcepb.Resource) string {
 }
 
 func sortedAttributes(attrs []*commonpb.KeyValue) []*commonpb.KeyValue {
-	sort.Slice(attrs[:], func(i, j int) bool {
-		return attrs[i].Key < attrs[j].Key
+	slices.SortFunc(attrs, func(a, b *commonpb.KeyValue) int {
+		if a.Key == b.Key {
+			return 0
+		}
+		if a.Key < b.Key {
+			return -1
+		}
+		return 1
```

Comment:
```suggestion
		return cmp.Compare(a.Key, b.Key)
```

Should we use `cmp.Compare` for readability?